Poofreading of feature/API-5352-agent-login-away (#605)
livechat/livechat-public-docs: content/messaging/agent-chat-api/v3.2/rtm-reference/index.mdx

| `application`                       | No       | `object`  |                                                                                                                                  |
| `application.name`                  | No       | `string`  | Application name                                                                                                                 |
| `application.version`               | No       | `string`  | Application version                                                                                                              |
| `away` __*__                        | No       | `bool`    | When `true`, the connection is set to the `away` state. Defaults to `false`.  |

> __*__ You can use the `away` param to **prevent assigning chats** to Agents with `not_accepting_chats` **after random reconnections**. When an Agent logs in with `away: true`, the connection is immediately recognized

Reviewer: >**prevent assigning chats** to Agents with `not_accepting_chats` **after random reconnections**

this might suggest that the chat assignment can happen even when the agent set `not_accepting_chats` with `set-routing-status` which is not the case

how about this?
__*__ You can use the `away` param to  **prevent assigning chats** to Agents **after random reconnections** when their status was set to `not_accepting_chats` by the auto-away feature.

### Set Away Status

> Sets an Agent's connection to the `away` state. You can use this method to manipulate the Agent's status. The method works per connection - all connections an Agents has (desktop, mobile, etc) must be in the `away` state. **Only then**, the Agent's status will change to `not_accepting_chats`.

Reviewer: ```suggestion
Sets an Agent's connection to the `away` state. You can use this method to manipulate the Agent's status. The method works per connection - all connections an Agent has (desktop, mobile, etc) must be in the `away` state for the Agent's status to be changed to `not_accepting_chats`.
```

Author: Hah, believe it or not, that was my first version. I changed it to avoid confusion by the "to be changed" structure, but maybe it looks like it should stay :)

### Set Routing Status

> Changes the status of an Agent or a Bot Agent. It sets the status for the session duration. Once the session ends, the status gets back to what's in the global setttings.

Reviewer: ```suggestion
Changes the status of an Agent or a Bot Agent. It sets the status for the session duration. Once the session ends, the status gets back to what's in the agent's settings.
```

> Unlike [**Set Away Status**](#set-away-status), which is another mechanism of status manipulation, **Set Routing Status** is indepenedent of the number of connections (desktop, mobile, etc).

Reviewer: ```suggestion
Unlike [**Set Away Status**](#set-away-status), which is another mechanism of status manipulation, **Set Routing Status** is independent of the number of connections (desktop, mobile, etc).
```
